alignak/log.py

```diff
@@ -216,19 +216,19 @@ def register_local_log(self, path, level=None, purge_buffer=True):
         # Todo : Do we need this now we use logging?
         return handler.stream.fileno()
 
-    def set_human_format(self, on=True):
+    def set_human_format(self, boolean=True):
```

Comment:
I think it's not a good idea to name variable 'boolean', perhaps name it 'human'